Retrieved from
  http://sources.redhat.com/cgi-bin/cvsweb.cgi/libc/sysdeps/powerpc/powerpc32/Makefile.diff?r1=1.4&r2=1.5&cvsroot=glibc
  http://sources.redhat.com/cgi-bin/cvsweb.cgi/libc/sysdeps/powerpc/powerpc64/Makefile.diff?r1=1.2&r2=1.3&cvsroot=glibc
Fixes
 /tmp/ccEVhqgD.s:279: Error: Unrecognized opcode: `stvx'
 /tmp/ccEVhqgD.s:288: Error: Unrecognized opcode: `lvx'
 make[2]: *** [powerpc-7450-linux-gnu/gcc-3.4.0-glibc-2.3.2/build-glibc/dlfcn/eval.os] Error 1
 make[2]: Leaving directory `powerpc-7450-linux-gnu/gcc-3.4.0-glibc-2.3.2/glibc-2.3.2/dlfcn'
 make[1]: *** [dlfcn/others] Error 2
 make[1]: Leaving directory `powerpc-7450-linux-gnu/gcc-3.4.0-glibc-2.3.2/glibc-2.3.2'
 make: *** [all] Error 2


Revision 1.5 Wed Oct 22 21:30:02 2003 UTC (7 months, 1 week ago) by roland
Branch: MAIN
CVS Tags: HEAD
Changes since 1.4: +0 -3 lines
Diff to previous 1.4 (colored)

2003-10-22  Steven Munroe  <sjmunroe@us.ibm.com>

	* sysdeps/powerpc/powerpc32/Makefile (+cflags, asm-CPPFLAGS): Don't
	append options to these.
	* sysdeps/powerpc/powerpc64/Makefile (+cflags, asm-CPPFLAGS): Likewise.

===================================================================
RCS file: /cvs/glibc/libc/sysdeps/powerpc/powerpc32/Makefile,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -r1.4 -r1.5
--- libc/sysdeps/powerpc/powerpc32/Makefile	2003/01/27 21:02:01	1.4
+++ libc/sysdeps/powerpc/powerpc32/Makefile	2003/10/22 21:30:02	1.5
@@ -1,8 +1,5 @@
 # Powerpc32 specific build options.
 
-+cflags += -Wa,-mppc -mpowerpc
-asm-CPPFLAGS += -Wa,-mppc
-
 ifeq ($(with-fp),no)
 +cflags += -msoft-float
 sysdep-LDFLAGS += -msoft-float
===================================================================
RCS file: /cvs/glibc/libc/sysdeps/powerpc/powerpc64/Makefile,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- libc/sysdeps/powerpc/powerpc64/Makefile	2002/09/27 19:44:16	1.2
+++ libc/sysdeps/powerpc/powerpc64/Makefile	2003/10/22 21:30:03	1.3
@@ -1,9 +1,6 @@
 # Powerpc64 specific build options.
 # this is ./sysdeps/powerpc/powerpc64/Makefile
 
-+cflags += -Wa,-mppc64 -mpowerpc64
-asm-CPPFLAGS += -Wa,-mppc64
-
 # Each TOC entry takes 8 bytes and the TOC holds up to 2^16 bytes,
 # or 8192 entries.
 # If -fpic is not specified, the latest gcc-3.2.1 now generates
